helm/nessie/templates/deployment.yaml

```diff
@@ -242,7 +245,7 @@ spec:
             failureThreshold: 3
             httpGet:
               path: /q/health/ready
-              port: 19120
+              port: 9000
```

Comment:
Technically, a successful check on port `9000` does not guarantee that the REST endpoints are ready on port `19120`, but I guess we can live with that risk.

Reply:
There is no real risk.
See `io.quarkus.vertx.http.runtime.VertxHttpRecorder#doServerStart` - the main server is started _before_ the _management_ server.